### PR TITLE
fix(runner): sync remote runner uri schema with runner_app

### PR DESCRIPTION
**Pull request by larme**
currently our runner_app and remote runner handler does not agree with how to handle `__call__` method. We need to sync them for runners using `__call__` method

- https://github.com/bentoml/BentoML/blob/main/bentoml/_internal/server/runner_app.py#L101
- https://github.com/bentoml/BentoML/blob/main/bentoml/_internal/runner/runner_handle/remote.py#L151

## Checklist:

- [ ] My code follows the bentoml code style, both `make format` and
  `make lint` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

**Review by sauyon — APPROVED** [review]
LGTM

**Comment by sauyon** [comment]
Actually, do we want a trailing slash on that API call or do we not care?

**Comment by sauyon** [comment]
Merging because this is relatively important.

File: bentoml/_internal/runner/runner_handle/remote.py [hunk 1]
```diff
@@ -147,8 +147,13 @@ async def async_run_method(
...
         multipart = payload_params_to_multipart(payload_params)
         client = self._get_client()
+
+        method_name = __bentoml_method.name
+        if __bentoml_method.name == "__call__":
+            method_name = ""
+
         async with client.post(
-            f"{self._addr}/{__bentoml_method.name}",
+            f"{self._addr}/{method_name}",
```
Comment (parano) [inline bentoml/_internal/runner/runner_handle/remote.py:156]: could we handle both "/__call__" and "/" on the server side? @bojiang @larme

Comment (bojiang) [reply bentoml/_internal/runner/runner_handle/remote.py:156]: Then what if there was a `call` on the runnable?

Comment (parano) [reply bentoml/_internal/runner/runner_handle/remote.py:156]: @bojiang oh I mean `/__call__`

Comment (bojiang) [reply bentoml/_internal/runner/runner_handle/remote.py:156]: @parano ah I agree. Will open a PR for that

**Pull request merged**